config_src/coupled_driver/ocean_model_MOM.F90

```diff
@@ -802,6 +806,8 @@ subroutine initialize_ocean_public_type(input_domain, Ocean_sfc, diag, maskmap,
   Ocean_sfc%v_surf(:,:)  = 0.0  ! time averaged v-current (m/sec)  passed to atmosphere/ice models
   Ocean_sfc%sea_lev(:,:) = 0.0  ! time averaged thickness of top model grid cell (m) plus patm/rho0/grav
   Ocean_sfc%frazil(:,:)  = 0.0  ! time accumulated frazil (J/m^2) passed to ice model
+  Ocean_sfc%melt_potential  = 0.0  ! time accumulated melt potential (J/m^2) passed to ice model
```

Comment:
This is an array syntax initialization statement. The MOM6 code style guide ( https://github.com/NOAA-GFDL/MOM6/wiki/Code-style-guide ) requires the addition of '(:,:)' to make it clear that this is this initialization of a 2-d array, so this line should become `Ocean_sfc%melt_potential(:,:) = 0.0 ! ...' , and similarly on the next line.